Forta-Bots-Challenge-1/src/agent.spec.ts

```diff
@@ -0,0 +1,282 @@
+        description: `Bot has been updated by ${NETHERMIND_FORTA_DEPLOYER_ADDRESS}`,
+        alertId: "FORTA-2",
+        severity: FindingSeverity.Low,
+        type: FindingType.Info,
+        metadata: {
+          agentID: mockDeploymentTxOne[0].toString(),
+          metadata: mockDeploymentTxOne[2],
+          chainIDs: mockDeploymentTxOne[3].toString(),
+        },
+      }),
+      expect.objectContaining({
+        name: "Nethermind Forta Bot Updated",
+        description: `Bot has been updated by ${NETHERMIND_FORTA_DEPLOYER_ADDRESS}`,
+        alertId: "FORTA-2",
+        severity: FindingSeverity.Low,
+        type: FindingType.Info,
+        metadata: {
+          agentID: mockDeploymentTxTwo[0].toString(),
+          metadata: mockDeploymentTxTwo[2],
+          chainIDs: mockDeploymentTxTwo[3].toString(),
+        },
+      }),
+    ]);
+  });
+  it("returns empty findings if there is a non-deploy or update call to the registry from Nethermind", async () => {
+    mockTxEvent
+      .setFrom(NETHERMIND_FORTA_DEPLOYER_ADDRESS)
+      .setTo(FORTA_REGISTRY_ADDRESS)
+      .addTraces({
+        function: FALSE_ABI.getFunction("destroyAgent"),
+        from: NETHERMIND_FORTA_DEPLOYER_ADDRESS,
+        to: FORTA_REGISTRY_ADDRESS,
+        arguments: mockDeploymentTxOne,
+      });
+    const findings = await handleTransaction(mockTxEvent);
+    expect(findings).toStrictEqual([]);
+  });
+  it("returns empty findings if there is a bot deployed from a different address", async () => {
+    const testAddress = createAddress("0x01");
+    mockTxEvent
+      .setFrom(testAddress)
+      .setTo(FORTA_REGISTRY_ADDRESS)
+      .addTraces({
+        function: AGENT_ABI.getFunction("createAgent"),
+        from: testAddress,
+        to: FORTA_REGISTRY_ADDRESS,
+        arguments: mockDeploymentTxOne,
+      });
+    const findings = await handleTransaction(mockTxEvent);
+    expect(findings).toStrictEqual([]);
+  });
+  it("returns correct findings if there is a mix of update, create and different function calls from Nethermind", async () => {
+    mockTxEvent
+      .setFrom(NETHERMIND_FORTA_DEPLOYER_ADDRESS)
+      .setTo(FORTA_REGISTRY_ADDRESS)
+      .addTraces({
+        function: FALSE_ABI.getFunction("destroyAgent"),
+        from: NETHERMIND_FORTA_DEPLOYER_ADDRESS,
+        to: FORTA_REGISTRY_ADDRESS,
+        arguments: mockDeploymentTxOne,
+      })
+      .addTraces({
+        function: AGENT_ABI.getFunction("createAgent"),
+        from: NETHERMIND_FORTA_DEPLOYER_ADDRESS,
+        to: FORTA_REGISTRY_ADDRESS,
+        arguments: mockDeploymentTxOne,
+      })
+      .addTraces({
+        function: AGENT_ABI.getFunction("updateAgent"),
+        from: NETHERMIND_FORTA_DEPLOYER_ADDRESS,
+        to: FORTA_REGISTRY_ADDRESS,
+        arguments: [mockDeploymentTxTwo[0], mockDeploymentTxTwo[2], mockDeploymentTxTwo[3]],
+      });
+    const findings = await handleTransaction(mockTxEvent);
+    expect(findings).toStrictEqual([
```

Comment:
```suggestion
      });
      
    const findings = await handleTransaction(mockTxEvent);
    
    expect(findings).toStrictEqual([
```
Spacing. In general, spacing is important, as it makes it for use (reviewers), to read.